package.json

```diff
@@ -2,27 +2,32 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "dependencies": {
-    "@firebase/firestore": "~0.3.4",
     "axios": "~0.17.1",
-    "expo": "^28.0.0",
-    "firebase": "~5.0.4",
+    "expo": "^36.0.0",
+    "expo-facebook": "~8.0.0",
+    "expo-image-picker": "~8.0.1",
+    "expo-linear-gradient": "~8.0.0",
+    "expo-location": "~8.0.0",
+    "expo-permissions": "~8.0.0",
+    "expo-web-browser": "~8.0.0",
+    "firebase": "~7.7.0",
     "ionicons": "~3.0.0",
-    "react": "16.3.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-28.0.0.tar.gz",
+    "react": "16.9.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz",
     "react-native-collapsible": "~0.10.0",
-    "react-native-elements": "~1.0.0-beta4",
+    "react-native-elements": "~1.2.7",
```

Comment:
Had to update RNE. Note that the API calls are probably slightly different, and our look-and-feel looks broken in several places. Created GH-139 to track this.